### PR TITLE
Feature/63/candidate request component

### PR DESCRIPTION
**Pull request by Nicolai-boiko**
изображение

### CODE REVIEW

File: src/app/components/candidate-request/candidate-request.component.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class CandidateRequestComponent implements OnInit {
   onSubmit(): void {
     if (this.registrationForm.valid) {
       console.log(this.registrationForm.value);
-      this.toastr.success('Вы успешно отправили форму', 'Принято')
+      this.toastr.success('Вы успешно отправили форму', 'Принято');
```
Comment (mhasilau) [inline src/app/components/candidate-request/candidate-request.component.ts:44]: message is only on russian?

File: src/app/components/candidate-request/candidate-request.component.ts [hunk 2]
```diff
@@ -1,9 +1,48 @@
...
   selector: 'app-candidate-request',
   templateUrl: './candidate-request.component.html',
   styleUrls: ['./candidate-request.component.scss'],
 })
-export class CandidateRequestComponent {}
+export class CandidateRequestComponent implements OnInit {
+  constructor(private toastr: ToastrService) {}
+  registrationForm: FormGroup;
+
+  cities: string[] = ['Moscow', 'New York', 'Minsk'];
```
Comment (ozhurova) [inline src/app/components/candidate-request/candidate-request.component.ts:14]: Is the API going to be used here?

File: src/assets/locales/en.json [hunk 3]
```diff
@@ -39,5 +39,29 @@
 		"invalidStartDate": "Invalid start date",
 		"name": "Sandbox",
 		"period": "Sandbox period"
+	},
+	"request": {
+		"header": "Candidate request",
+		"firstName": "First Name",
+		"secondName": "Second Name",
```
Comment (ozhurova) [inline src/assets/locales/en.json:46]: Maybe Last Name or surname ?

File: src/app/components/candidate-request/candidate-request.component.ts (hunk 2 above)
Comment (Nicolai-boiko) [reply src/app/components/candidate-request/candidate-request.component.ts:14]: changed to simple input

File: src/app/components/candidate-request/candidate-request.component.ts (hunk 1 above)
Comment (Nicolai-boiko) [reply src/app/components/candidate-request/candidate-request.component.ts:44]: Now its only in english

**Review by vstrabko — APPROVED** [review]

File: src/app/components/candidate-request/candidate-request.component.ts (hunk 1 above)
Comment (Nicolai-boiko) [reply src/app/components/candidate-request/candidate-request.component.ts]: done

File: src/app/components/candidate-request/candidate-request.component.scss [hunk 4]
```diff
@@ -1,2 +1,71 @@
 @import '../../shared/constants/colors.scss';
 @import '../../shared/constants/otherStyles.scss';
+
+form {
+  padding: 5px 15px;
+}
+
+.sandbox > * {
```
Comment (rastorc3v) [inline src/app/components/candidate-request/candidate-request.component.scss:8]: * selector is not a good idea, it affects performance. You can leave it like this for now, but next time use classes or something else.

**Review by mhasilau — APPROVED** [review]

**Review by rastorc3v — APPROVED** [review]

File: src/app/components/candidate-request/candidate-request.component.scss (hunk 4 above)
Comment (Nicolai-boiko) [reply src/app/components/candidate-request/candidate-request.component.scss:8]: ok, thx

**Pull request merged**